backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -204,10 +213,38 @@ export class ContributionResolver {
 
     // all possible cases not to be true are thrown in this function
     validateContribution(creations, amount, creationDateObj)
+
+    const contributionMessage = ContributionMessage.create()
+    contributionMessage.contributionId = contributionId
+    contributionMessage.createdAt = contributionToUpdate.updatedAt
```

Comment:
why is this needed? is updatedAt not set when the message is created?

Reply:
See the issue. There is explained why, @ulfgebhardt

Reply:
@ulfgebhardt & @Mogge
updatedAt is setten after the creation of the message, and the issue was saying createdAt of the message is updatedAt of the contribution if it has one else the createdAt.
After reflecting it I am not sure why and if it is not better to set the value of now on createdAt since the message is created now. If we let the logic so we could have this HISTORY message in between some old messages